lib/node_modules/@stdlib/math/base/special/lcmf/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+    "options": {
+      "task": "build"
+    },
+    "fields": [
+      {
+        "field": "src",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "include",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "libraries",
+        "resolve": false,
+        "relative": false
+      },
+      {
+        "field": "libpath",
+        "resolve": true,
+        "relative": false
+      }
+    ],
+    "confs": [
+      {
+        "task": "build",
+        "src": [
+          "./src/main.c"
+        ],
+        "include": [
+          "./include"
+        ],
+        "libraries": [],
+        "libpath": [],
+        "dependencies": [
+          "@stdlib/math/base/napi/binary",
+          "@stdlib/math/base/assert/is-nan",
+          "@stdlib/math/base/special/gcd"
```

Comment:
These and the dependencies below must be changed to their single-precision equivalents.